CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/thirdparty" ${CMAKE_MODULE_PA
 
 # Build options
 option(RAJA_ENABLE_OPENMP "Build OpenMP support" On)
+option(RAJA_ENABLE_TBB "Build TBB support" On)
```

Comment:
Our other options shouldn't fallback like this (and definitely won't with BLT), i.e. if you ask for an option and we can't find it, CMake should fail...